src/components/ReviewSwap.tsx

```diff
@@ -116,6 +117,10 @@ function ReviewSwap({ onClose, onConfirm, data }: Props): ReactElement {
               {formatSlippageToString(slippageSelected, slippageCustom)}%
             </span>
           </div>
+          <div className="row">
+            <span className="title">{t("deadline")}</span>
+            <span className="value floatRight">{transactionDeadline} min</span>
```

Comment:
should use the translation for `min` here

Reply:
changed to translation lib syntax for minutes